Projects/Bounties/README.md

```diff
@@ -15,11 +15,14 @@
 1. Requirements (acceptance criteria) are managed by the Gatekeeper.
 1. An estimate (from the table below) is provided by the Keymaster.
 1. The Gatekeeper and Keymaster SHOULD negotiate requirements and estimates until they come to a common understanding.
+1. The Treasurer SHOULD review the bounty's market value against comparable work in other markets.
 1. The Treasurer is responsible for publishing a unique BTC address for each task.
+1. The Treasurer is responsible for periodically adjusting bounty sizing estimate table.
```

Comment:
We'll have to be careful about keeping redundant rules in sync. The [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) applies to policy as well.